config.py

```diff
@@ -0,0 +1,2 @@
+# put your ngrok token below
```

Comment:
`config.py` is not required since ngrok is not being added, please remove this file
also
bad security practice : `you have leaked your ngrok token`